app/javascript/components/util/DeleteDialog.js

```diff
@@ -1,6 +1,9 @@
 import React from 'react';
```

Comment:
Please update the tests for this component to include one for showing the symptom onset stuff

Reply:
https://github.com/SaraAlert/SaraAlert/pull/861/commits/ea0697f83c29320b8930fb7a5dcf63015ea342fb#diff-640be94f511d6918e9e1e33925a272262c842e30e77a89b05a1555ee5dae352cR135